src/amberscript/parser.cc

```diff
@@ -60,14 +60,18 @@ Result Parser::Parse(const std::string& data) {
       r = ParseRun();
     } else if (tok == "SHADER") {
       r = ParseShaderBlock();
+    } else if (tok == "CLEAR") {
```

Comment:
I think this was a bad merge, clear was moved up to line 53 to put these in sorted order (can you move COPY up there as well?)